packages/flutter/test/widgets/sensitive_content_test.dart

```diff
@@ -0,0 +1,1414 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  // Default content sensitivity setting for testing.
+  final int defaultContentSensitivitySettingId = ContentSensitivity.autoSensitive.id;
```

Comment:
I only tested with this as the default as of right now because I couldn't convince myself that there was greater value in testing other defaults besides the method channel `setContentSensitivity` method getting called. If reviewers think that is valuable, I can make new test files to test the other modes as defaults (I think I have to do this because of the singleton pattern I used for `SensitiveContentSetting`).

Reply:
I think it's not a big deal, in the code it looks like there is no special behavior for different default values. If you wanted to though, you could quickly do it by looping over ContentSensitivity.values and running the same test(s) for each value.